pkg/config/config_test.go

```diff
@@ -14,38 +14,51 @@
 package config_test
 
 import (
+	"bytes"
 	"flag"
 	"os"
 	"testing"
 
 	"github.com/aws/aws-node-termination-handler/pkg/config"
 	h "github.com/aws/aws-node-termination-handler/pkg/test"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
 )
 
+var envForTest = map[string]string{}
+
 func resetFlagsForTest() {
 	flag.CommandLine = flag.NewFlagSet(os.Args[0], flag.ContinueOnError)
 	os.Args = []string{"cmd"}
+	for key := range envForTest {
+		os.Unsetenv(key)
+	}
+}
+
+func setEnvForTest(key string, val string) {
+	os.Setenv(key, val)
+	envForTest[key] = val
 }
 
 func TestParseCliArgsEnvSuccess(t *testing.T) {
```

Comment:
Is `ENABLE_SQS_TERMINATION_DRAINING` omitted purposefully?

Reply:
Not really. I don't think we need to add every new flag in these tests, honestly they're not great tests, but I'll add the enable flag since it's all we have right now.